components/org.wso2.carbon.msf4j.interceptor.common/pom.xml

```diff
@@ -40,6 +40,10 @@
             <groupId>org.wso2.msf4j</groupId>
```

Comment:
I think you should not use org.wso2.carbon.msf4j.interceptor.common
but use org.wso2.carbon.**analytics**.msf4j.interceptor.common

Reply:
same for the packages

Reply:
Fixed in 497ede4